src/Extensions/CwpSearchBoostExtension.php

```diff
@@ -52,6 +46,23 @@ public function updateCMSFields(FieldList $fields)
         $boostField = TextareaField::create('SearchBoost', $boostTitle)
             ->setRightTitle($boostNote)
             ->setDescription($boostDescription);
-        $fields->insertBefore('MetaDescription', $boostField);
+
+        if ($meta = $fields->fieldByName('Root.Main.Metadata')) {
+            // Rename metafield if it exists
+            $meta->setTitle($pageInfoTitle);
+            $fields->insertBefore('MetaDescription', $boostField);
```

Comment:
This line would previously be run regardless of whether the Metadata field exists, but it's now only run when it does exist. Perhaps you could move this back up to be underneath the `$boostField` definition